spec/controllers/events_controller_spec.rb

```diff
@@ -170,6 +170,14 @@
         expect(assigns(:email)).to have_attributes(hide_recipients: false, recipients: @event.email_adresses_of_rejected_applicants, reply_to: 'workshop.portal@hpi.de', subject: '', content: '')
       end
     end
+
+    describe "GET #accept_all_applicants" do
+      it "should accept all application letters" do
```

Comment:
I dont think this is something you should test in a controller test. That belongs into model & feature tests.
Instead it should check whether the variables are set, if it renders the correct template/redirects correctly, etc.